### PR TITLE
Localize decimal separator

### PR DESCRIPTION
**Pull request by eVoloshchak**
### Details
This change allows user to use comma (`,`) instead of a dot (`.`) as a decimal separator on `WorkspaceReimburseView` and `IOUAmountPage` when using Spanish language.

### Fixed Issues

$ https://github.com/Expensify/App/issues/9497

### Tests

1. Have English language enabled
2. Click `FAB > Request money`
3. Verify that there is `.` button at the bottom left corner of the onscreen keyboard
4. Verify that it's possible to enter a decimal number with two digits after the **dot**
5. Open `Settings > Workspace > Reimburse expenses`
6. Verify that it's possible to enter a decimal number with three digits after the **dot** into `Rate` input
7. Change language to Spanish
8. Click `FAB > Request money`
9. Verify that there is `,` button at the bottom left corner of the onscreen keyboard
10. Verify that it's possible to enter a decimal number with two digits after the **comma**
11. Open `Settings > Workspace > Reimburse expenses`
12. Verify that it's possible to enter a decimal number with three digits after the **comma** into `Rate` input
#### Contributor (PR Author) Checklist
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/docs/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [ ] MacOS / Desktop
- [x] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/docs/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/docs/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/docs/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] Any functional components have the `displayName` property
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose and it is
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn’t already exist
    - [x] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.

<details>
<summary><h4>PR Reviewer Checklist</h4></summary>

- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/docs/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/docs/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/docs/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/docs/PR_REVIEW_GUIDELINES.md)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] Any functional components have the `displayName` property
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn’t already exist
    - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.

</details>

### QA Steps

1. Have English language enabled
2. Click `FAB > Request money`
3. Verify that there is `.` button at the bottom left corner of the onscreen keyboard
4. Verify that it's possible to enter a decimal number with two digits after the **dot**
5. Open `Settings > Workspace > Reimburse expenses`
6. Verify that it's possible to enter a decimal number with three digits after the **dot** into `Rate` input
7. Change language to Spanish
8. Click `FAB > Request money`
9. Verify that there is `,` button at the bottom left corner of the onscreen keyboard
10. Verify that it's possible to enter a decimal number with two digits after the **comma**
11. Open `Settings > Workspace > Reimburse expenses`
12. Verify that it's possible to enter a decimal number with three digits after the **comma** into `Rate` input

#### Web

https://user-images.githubusercontent.com/9059945/178724447-8b822aa2-eeb0-466a-9d5e-470f99a792de.mp4

https://user-images.githubusercontent.com/9059945/177964283-2787cf1a-62a1-462d-92cf-6a5bb162b241.mp4

#### Mobile Web

https://user-images.githubusercontent.com/9059945/177965086-ada5d846-0453-48d4-a48c-8af6d6a9e80d.mp4

#### iOS

https://user-images.githubusercontent.com/9059945/177965306-793c4550-7e58-4870-8ffe-accb8c6eee32.MP4

#### Android

https://user-images.githubusercontent.com/9059945/177964890-59fead34-64e4-485c-b51b-2526af4b9244.mp4

### CODE REVIEW

**Comment by Santhosh-Sellavel** [comment]
@eVoloshchak Check your web video from 00 to 06 secs. I see value getting lost temporarily. You are entering 233.32, it appears 33.32 for fraction of seconds.

**Review by sketchydroide — APPROVED** [review]

**Review by sketchydroide — CHANGES_REQUESTED** [review]
sorry I aproved while only seeing the code, but @Santhosh-Sellavel  is right the web video looks off

**Comment by eVoloshchak** [comment]
> @eVoloshchak Check your web video from 00 to 06 secs. I see value getting lost temporarily. You are entering 233.32, it appears 33.32 for fraction of seconds.

I think that my screen recorder is recording at low fps, what i see on the screen and on the video are not the same thing. I updated the web section with a screen capture shot on a phone camera

https://user-images.githubusercontent.com/9059945/178724790-01ed7d72-90d1-44f8-af20-11ee04df22b3.mp4

**Comment by Santhosh-Sellavel** [comment]
> > @eVoloshchak Check your web video from 00 to 06 secs. I see value getting lost temporarily. You are entering 233.32, it appears 33.32 for fraction of seconds.
> 
> I think that my screen recorder is recording at low fps, what I see on the screen and on the video are not the same thing. I updated the web section with a screen capture shot on a phone camera
> 
>  video_2022-07-13_14-33-58.mp4

Please update the screen record so that's a more reliable option, to ensure everything. I believe you can do that on macincloud also.

**Comment by eVoloshchak** [comment]
> Please update the screen record so that's a more reliable option, to ensure everything. I believe you can do that on macincloud also.

Sure! This is a screen recording from Windows machine, it seems to have more fps.

https://user-images.githubusercontent.com/9059945/178995567-801ae501-ed19-44dc-800c-32c4f67a53c0.mp4

**Comment by Santhosh-Sellavel** [comment]
Thanks, @eVoloshchak can you repeat the same steps in staging and share the recording, thanks!

> > Please update the screen record so that's a more reliable option, to ensure everything. I believe you can do that on macincloud also.
> 
> Sure! This is a screen recording from Windows machine, it seems to have more fps.
> 
>  1.New.Expensify.-.Google.Chrome.2022-07-14.15-44-32.mp4



> > Please update the screen record so that's a more reliable option, to ensure everything. I believe you can do that on macincloud also.
> 
> Sure! This is a screen recording from Windows machine, it seems to have more fps.
> 
>  1.New.Expensify.-.Google.Chrome.2022-07-14.15-44-32.mp4

**Comment by eVoloshchak** [comment]
> @eVoloshchak can you repeat the same steps in staging and share the recording. With that, we can rule out whether the behavior is caused by this PR or not thanks!

Of course

https://user-images.githubusercontent.com/9059945/179361099-c7da50e4-e381-4feb-b0a6-c7e14d663e20.mp4

**Comment by Santhosh-Sellavel** [comment]
#### PR Reviewer Checklist

- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] Any functional components have the `displayName` property
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn’t already exist
    - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.

**Review by Santhosh-Sellavel — APPROVED** [review]
Looks good and tests well @sketchydroide.

**Review by sketchydroide — APPROVED** [review]

**Comment by sketchydroide** [comment]
@eVoloshchak great job getting your first Expensify/App pull request over the finish line! I'll merge it assuming all the tests pass.

I know there's a lot of information in our contributing guidelines, so some points to take note of:

Once your PR is merged, you can be hired for another issue. Once you've completed a few issues, we may start hiring you for more than one issue at a time.

Once your PR is deployed to our staging servers, it will undergo quality assurance (QA) testing. If we find that this doesn't work as expected or causes a regression, you'll be responsible for fixing it. Typically we would revert this PR and give you another chance to create a similar PR without causing a regression. (I don't imagine this will happen with this PR, but it's something to be aware of)

Once your PR is deployed to production, we start a 7-day timer. After it has been on production for 7 days without causing any regressions, then we pay out the Upwork job.

So it might take a while before you're paid for your work, but we typically post multiple new jobs every day, so there's plenty of opportunity. I hope you've had a positive experience contributing to this repo!

File: src/components/BigNumberPad.js
Comment (parasharrajat) [inline src/components/BigNumberPad.js:66]: How is this change related to this PR? cc: @Santhosh-Sellavel

File: src/pages/workspace/reimburse/WorkspaceReimburseView.js
Comment (parasharrajat) [inline src/pages/workspace/reimburse/WorkspaceReimburseView.js:84]: use of `fromLocaleDigit` is wrong here as per your changes. Why? 

fromLocaleDigit accepts a localized char, `.` is not decimal in spanish so it should `this.props.fromLocaleDigit(',');`

**Review by parasharrajat — COMMENTED** [review]
I was checking some other PR and saw these changes to IOUAmountPage which I feel like very off from the planned design of this page. 

I tested the IOUAmountPage before this PR and it was accepting `,` instead of `.` for Spanish locale. 

Why do we have to change IOUAmountPage here? First of all, How is this related to this issue? 



cc: @Santhosh-Sellavel

File: src/pages/workspace/reimburse/WorkspaceReimburseView.js
Comment (eVoloshchak) [reply src/pages/workspace/reimburse/WorkspaceReimburseView.js:84]: > use of `fromLocaleDigit` is wrong here as per your changes. Why?


Thanks, that is a good catch. They return the same value when passing `.` and I missed it and used the wrong function on `IOUAmountPage`, while using the right one on `BigNumberPad`. I'll open a PR with fix for this after the discussion around the `BigNumberPad`.

File: src/components/BigNumberPad.js
Comment (parasharrajat) [reply src/components/BigNumberPad.js:66]: That is how it is supposed to work Internally the values will always be in English for eg. 
`22,23` is really `22.23` which will be saved to the backend. So it was working correctly. 

Manipulating this value here is not a good approach. These changes break the whole concept.

Comment (Santhosh-Sellavel) [reply src/components/BigNumberPad.js:66]: @parasharrajat 

We are not saving it as `22,23` or `22.33`. We are rounding it off after multiplying by a hundred, so it's saved as `2233`. I'm not sure about what we broke here?

cc: @sketchydroide

File: src/pages/workspace/reimburse/WorkspaceReimburseView.js
Comment (Santhosh-Sellavel) [reply src/pages/workspace/reimburse/WorkspaceReimburseView.js:84]: `.` is the grouping separator in Spanish, i.e `,` in English.

Easy to miss, thanks @parasharrajat! 

Need to update this @eVoloshchak!

cc: @sketchydroide

Comment (eVoloshchak) [reply src/pages/workspace/reimburse/WorkspaceReimburseView.js:84]: > Need to update this @eVoloshchak!

I can fix this in https://github.com/Expensify/App/pull/10028, would that be ok?

File: src/components/BigNumberPad.js
Comment (parasharrajat) [reply src/components/BigNumberPad.js:66]: by the backend, I mean internally not DB. Why do we have to change this piece of code? It does not look related to the issue.

File: src/pages/workspace/reimburse/WorkspaceReimburseView.js
Comment (parasharrajat) [reply src/pages/workspace/reimburse/WorkspaceReimburseView.js:84]: Please hold on to it. We are first trying to decide why are these changes needed at all.

File: src/components/BigNumberPad.js
Comment (eVoloshchak) [reply src/components/BigNumberPad.js:66]: > Why do we have to change this piece of code? It does not look related to the issue. Can anyone please help me understand this?

This keyboard is used only on `IOUAmountPage`, so it seemed logical to make it return the displayed symbol. 
Alternatively we can move this logic to `IOUAmountPage`, change [this](https://github.com/eVoloshchak/App/blob/bcf54e6d4809cf2c0fb7e6bd4fde75214b0186f8/src/pages/iou/steps/IOUAmountPage.js#L143) line to

```javascript
const amount = `${prevState.amount}${this.props.toLocaleDigit(key)}`;
```

Comment (eVoloshchak) [reply src/components/BigNumberPad.js:66]: > Do you think there was any bug on IOUAmountPage that you solved here? What was that? Issue talks about the problem with the rate field on the reimbursement form. 

But the screenshot in the issue description features `Request money` screen. I've asked about this a [couple](https://github.com/Expensify/App/issues/9497#issuecomment-1166304208) of [times](https://github.com/Expensify/App/issues/9497#issuecomment-1167763611), there must have been a miscommunication
> OK, i think i got confused, since the issue description is talking about the Rate input on Reimburce Expenses screen, while screenshot contains IOU Amount Page with an amount input
I'm still not sure which one of the two screens needs to be fixed



> Can we fix the rate field issue without touching IOUAmountPage?

Yes we can
I've checked new.expensify.com and it does indeed support comma for Spanish language on Request Money screen.
I'm guessing we need o revert this PR and clarify what the requirements for this issue are, it it's just for the rate field

Comment (parasharrajat) [reply src/components/BigNumberPad.js:66]: It seems yes issue details are incorrect. Unfortunately, it was not captured during the proposal review. As a suggestion, please try to get a full consensus before you start if you feel something is off. You can also suggest corrections to the requirements. 

Thanks for understanding. I will open this discussion on the issue so that this PR can be reverted and you can start new.

**Comment by mountiny** [comment]
@eVoloshchak and @parasharrajat the linked issue has been incorrectly linked:

`$ https://github.com/Expensify/App/issues/9497` the dollar sign was missing. Please, pay attention to this for next PRs. Thank you 🙇

**Comment by parasharrajat** [comment]
@Santhosh-Sellavel was C+ here.

**Comment by mountiny** [comment]
Sorry Rajat, saw you amongst the reviewers. @Santhosh-Sellavel  for the note above 🙇

**Comment by Santhosh-Sellavel** [comment]
🤦 Oops 

@mountiny Thanks for the note!

**Comment by puneetlath** [comment]
Heads up @parasharrajat @Santhosh-Sellavel @sketchydroide! Just FYI: this PR caused a bug here: https://github.com/Expensify/App/issues/11352

**Comment by parasharrajat** [comment]
I already asked to revert it https://github.com/Expensify/App/pull/9800#discussion_r925924728. we had a revert PR https://github.com/Expensify/App/pull/10129. 

There must be follow up PR. cc: @Santhosh-Sellavel as C+.

**Comment by Santhosh-Sellavel** [comment]
@puneetlath This PR was reverted not sure how it caused a regression!

**Comment by puneetlath** [comment]
Good point fellas. My bad lol. Will look further into it!

**Pull request merged**
